packages/portfolio-contract/src/pos-usdn.flows.ts

```diff
@@ -115,8 +116,12 @@ export const protocolUSDN = {
     );
 
     trace('executing', [msgSwap, msgLock].filter(Boolean));
-    const result = await ica.executeEncodedTx(protoMessages);
-    trace('supply result', result);
+    const ret = ica.executeEncodedTxWithMeta(protoMessages);
+    unwrapResultMeta(ret).then(
```

Comment:
that looks like a floating promise. I wonder why it's not a static error.

Reply:
it's not floating because it has both success and rejection handlers.

https://typescript-eslint.io/rules/no-floating-promises/